### PR TITLE
Update VisualQnA README.md for its workflow

### PR DESCRIPTION
**Pull request by louie-tsai**
## Description

Added a workflow for VisualQnA following what ChatQnA has. [ChatQnA flow](https://github.com/opea-project/GenAIExamples/tree/main/ChatQnA#architecture-and-deploy-details)

## Issues

 `n/a`.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

na

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

File: VisualQnA/README.md [hunk 1]
```diff
@@ -13,6 +13,59 @@ General architecture of VQA shows below:
...
+        direction LR
+        a([User Input Query]):::orchid
+        Ingest([Ingest data]):::orchid
+        UI([UI server<br>]):::orchid
+    end
+
+
+    LVM_gen{{LVM Service <br>}}
+    GW([VisualQnA GateWay<br>]):::orange
+    NG([Nginx MicroService]):::blue
```
[the remark this answers is not in the document]
Comment (louie-tsai) [reply VisualQnA/README.md:51]: I saw a nginx microservice as a proxy server in docker compose, and UI server depends it.
https://github.com/opea-project/GenAIExamples/blob/main/VisualQnA/docker_compose/intel/hpu/gaudi/compose.yaml#L76

should I remove it or put it somewhere else in the workflow?

[the remark this answers is not in the document]
Comment (lvliang-intel) [reply VisualQnA/README.md:51]: should be "User Input" --> Nginx svc ---> UI Server

[the remark this answers is not in the document]
Comment (louie-tsai) [reply VisualQnA/README.md:51]: here is the related change. Let me know whether it works for you or not.
image

[the remark this answers is not in the document]
Comment (louie-tsai) [reply VisualQnA/README.md:51]: @lvliang-intel 
let me know any modification needed.

**Comment by louie-tsai** [comment]
@lvliang-intel 
Could you help to review and merge the PR?

**Review by lvliang-intel — APPROVED** [review]

**Review by yinghu5 — APPROVED** [review]
LGTM

**Pull request merged**